README.md

```diff
@@ -1,32 +1,33 @@
-# Indeed Open Source Repository Template
+# Mariner
 
 ![OSS Lifecycle](https://img.shields.io/osslifecycle/indeedeng/default-template.svg)
```

Comment:
We should try to figure out what this should be

Reply:
who would have information on that?

Reply:
A search for "OSS Lifecycle" got me to the shields project, and to this page: 
https://shields.io/category/other

Reading more, it seems that if we put in a link to this project on github, it will pull information from the OSSMETADATA file, and display the correct current status. You can see a working example here: https://raw.githubusercontent.com/indeedeng/proctor/master/README.md